common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -100,6 +100,8 @@ public BrokerRequest brokerRequestFromAcquireTokenParameters(@NonNull final Inte
                 .scope(TextUtils.join(" ", parameters.getScopes()))
                 .redirect(parameters.getRedirectUri())
                 .clientId(parameters.getClientId())
+                .childRedirectUri(parameters.getChildRedirectUri())
```

Comment:
Not of a fan of the naming. Some places you've chosen a new name for the child i.e. the child and use existing parameter for the hub. Whereas other places you are using existing parameter for the child and use the new naming for the hub.

Reply:
Added reply to a similar comment abobe